### ISSUE TITLE
Wrong num_train_optimization_steps with gradient accumulation

### ISSUE THREAD

**Issue by e-bug** [facebookresearch/vilbert-multi-task#20]
Hey @vedanuj, it looks like there is a bug in the number of training steps when using gradient accumulation (L343):
https://github.com/facebookresearch/vilbert-multi-task/blob/72770f18688bb55e0a0eb5aaf389055eb3acdd6e/train_concap.py#L340-L344

as `args.train_batch_size` was already divided by `args.gradient_accumulation_steps` in L288:
https://github.com/facebookresearch/vilbert-multi-task/blob/72770f18688bb55e0a0eb5aaf389055eb3acdd6e/train_concap.py#L288

PS. This issue is also present in vilbert_beta (@jiasenlu)
